app/create/components/CreateSubscriptionForm.tsx

```diff
@@ -19,7 +19,13 @@ import { SATS_CURRENCY } from "lib/constants";
 type CreateSubscriptionFormData = Omit<
   CreateSubscriptionRequest,
   "nostrWalletConnectUrl" | "payerData" | "sleepDuration"
-> & { payerName: string; timeframe: Timeframe; timeframeValue: string };
+> & {
+  payerName: string;
+  timeframe: Timeframe;
+  timeframeValue: string;
+  useCron: boolean;
```

Comment:
I don't think `useCron` should be part of the form data - can it be a normal `React.useState?`